gaea/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
```

Comment:
Since you are the package author, why not [bundle](http://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html) types?